### DISCUSSION TITLE
ILS/LOC Mode/APPR mode

### DISCUSSION THREAD

**Discussion by IlanMoshe** [flybywiresim/aircraft#3437] [Help]
Hello, 

When ATC gives me a radar vectoring to capture the ILS and not to follow the STAR or the transition published, the last clearance should be like:
 "AFR200, fly heading 070, cleared ILS RWY 04 L".
So, as a pilot, I set heading 070, and press on LOC or APPR and have to wait to the capture of the ILS. 
But, the A320 NX behaves quite differently, indeed after be stable on 070 and pressed on LOC/APPR mode, it seems that it starts to capture the ILS even though it didn't cross the radial. 

Someone can help me with this.

Thank you

**Comment by riveroalbert** [comment]
As far as I'm concerned LOC mode will calculate the intercept heading, (i think arround 30 degrees) which might differ from the ILS intercept heading given by the ATC. So I guess to avoid overshooting the ils, FCU makes its own calculation. Could it be that?

**Reply by IlanMoshe** [reply]
Yes, but it should not. 
You should arm LOC once you are on a closing heading and around 30 degrees to the localiser as you said. But as far as I know about the A320, only when crossing the radial the airbus start to intercept the localizer and not before.

**Reply by ExampleWasTaken** [reply]
Pressing the LOC pb tells the Airbus to intercept the Localizer at an optimal intercept heading. So based on your description that is expected behavior.

**Reply by IlanMoshe** [reply]
I'm not agree with "at an optimal intercept heading.".
The airbus has to intercept the localizer at a present heading. If I'm flying heading 070 to intercept the ILS 04 and press on LOC/APPR, the airbus will turn only when crossing the localizer. If I'm some miles before the localizer, flying 070 and press on LOC, the airbus should continue to fly heading 070.

**Reply by ExampleWasTaken** [reply]
Any references from the real aircraft?

**Reply by ExampleWasTaken** [reply]
@IlanMoshe [here](https://youtu.be/32q0xHybKmA?t=1079) is a reference that might help you understand what I mean.

**Reply by IlanMoshe** [reply]
"You can use the navigation mode in combination with the heading function to fly an assigned heading to intercept a course. The procedure illustrated in Figure 4-13 takes advantage of the ability to arm the navigation mode while the heading mode is engaged." P11

"As the autopilot captures the course, the heading function automatically disengages, and the navigation function switches from armed to engaged." P12 (picture)

From : https://www.faa.gov/regulations_policies/handbooks_manuals/aviation/advanced_avionics_handbook/media/aah_ch04.pdf

So if I understood correctly, only when capturing, the airbus LOC approach became "engaged" and therefore has to start the turn.

**Reply by IlanMoshe** [reply]
I'm not talking about the case showed on this tutorial ;)
He pressed on LOC during the final turn .. I'm talking about pressing LOC when I'm flying, stable on heading 070 to intercept ILS 04 and I'm some miles before the localizer for example.

**Reply by ExampleWasTaken** [reply]
That file does not refer to the A20N.

**Reply by IlanMoshe** [reply]
I think we have to separate the phase when the LOC is **armed** and the phase when the LOC mode became **engaged**.
When you press on LOC, you're **arm**ing the LOC but it is not necessary and immediately became **engaged**.
It will pass as engaged once it will intercept the localizer.

Think the APP controller should create the sequence via one point which is not on the final approach path. 
So, if he gave you the instruction to fly HDG 070 to intercept the ILS 04L, you must continue to fly 070 until the interception, meaning arm the LOC should not deviate the airbus from 070 until the interception.

**Reply by ExampleWasTaken** [reply]
Alright, I think I know what you mean. 
Pressing the LOC button will *arm* the LOC mode. Following that the FMA display LOC blue. The airbus will then start to fly in a manner that allows it to intercept the localizer at an optimal heading/track. Once it's close enough to the localizer it will change to LOC* meaning it starts to capture the localizer. Once established the FMA displays LOC.

If you are not yet cleared to intercept the LOC you don't press the button yet. Same goes for the APPR pb.
